R/driver.R

```diff
@@ -25,17 +25,27 @@ run_chunk <- function(chunk, all_data) {
 #' @param chunk_data Data produced by chunk
```

Comment:
These param documentations should include the type of the data structure.  I assumed a list in some of my other comments, but maybe I'm confused and the actual structure is something else.